### PR TITLE
Add some CRAM history and acknowledgements

### PR DESCRIPTION
**Pull request by jkbonfield**
Getting the ball rolling...

### CODE REVIEW

**Comment by jkbonfield** [comment]
Added a small tweak to list other contributors by name;  just a brief alphabetical list.  This is everyone who has authored a git commit or is the author of an issue/PR which is more than a simple question (eg informing us of incorrect or unclear text).

(I propose merging next week regardless of lack of review given it's not a normative part of the specification so it cannot "break the spec".)

File: CRAMv3.tex [hunk 1]
```diff
@@ -2970,4 +2970,111 @@ \subsection{\textbf{Choosing the container size}}
...
+specification.
+
+\item Joel Thibault (Broad Institute):
+previous maintainer of CRAM specification.
+
+\item Chris Norman (Broad Institute):
+previous maintainer of CRAM specification and worked on
+the HTSJDK implementation.
+
+\item Robert Buels (UoC, Berkeley): First JavaScript implementation of CRAM
```
Comment (jmarshall) [inline CRAMv3.tex:3069]: I was confused what `UoC` indicated here — usual spelling out of UCB I think is
either `University of California, Berkeley` or `UC Berkeley`.

File: CRAMv3.tex [hunk 2]
```diff
@@ -2970,4 +2970,111 @@ \subsection{\textbf{Choosing the container size}}
...
+\end{itemize}
+
+\subsubsection*{CRAM 3.0: 2014}
+
+Primarily this is an optimisation of size and speed.
+
+\begin{itemize}
+\item Inclusion of lzma compression library.
+\item Inclusion of the custom rANS Order-0 and Order-1 entropy encoders.
+\item Checksums added to all file format structures to insure data integrity.
```
Comment (jmarshall) [inline CRAMv3.tex:3043]:
```suggestion
\item Checksums added to all file format structures to ensure data integrity.
```

**Review by jmarshall — COMMENTED** [review]
Looks good. Couple of specific comments, and a few general formatting minor things: a few missing full stops; capitalise “Previous maintainer …” in a couple of bullets (as the other items are capitalised).

File: CRAMv3.tex [hunk 3]
```diff
@@ -2970,4 +2970,111 @@ \subsection{\textbf{Choosing the container size}}
...
+\begin{itemize}
+\item EOF blocks are added in order to spot truncated files.
+\end{itemize}
+
+\subsubsection*{CRAM 3.0: 2014}
+
+Primarily this is an optimisation of size and speed.
+
+\begin{itemize}
+\item Inclusion of lzma compression library.
```
Comment (jmarshall) [inline CRAMv3.tex:3041]:
```suggestion
\item Inclusion of LZMA compression library.
```

**Pull request merged**